modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -56,3 +130,31 @@ If the app update was successful, the app no longer appears in the *Update Avail
 
 If the app update was unsuccessful, the app remains in the *Update Available* tab.
 In this case, the app continues to run correctly on the previous runtime version and no action is required.
+
+== Update Multiple Apps to the Latest Runtime Release 
```

Comment:
Also a peer of monthly date patch updates and manual runtime version updates? Or is this actually subordinate to manual runtime version updates?

Reply:
Made both subsections to a new == level.